osu.Game/Screens/Edit/Compose/Components/SelectionRotationHandler.cs

```diff
@@ -13,9 +13,10 @@ namespace osu.Game.Screens.Edit.Compose.Components
     public partial class SelectionRotationHandler : Component
     {
         /// <summary>
-        /// Whether the rotation can currently be performed.
+        /// Whether rotation anchored by the selection origin can currently be performed.
+        /// This is in constrast to rotation anchored by the entire field.
         /// </summary>
-        public Bindable<bool> CanRotate { get; private set; } = new BindableBool();
+        public Bindable<bool> CanRotateSelectionOrigin { get; private set; } = new BindableBool();
```

Comment:
this can't just be renamed to "selection origin", there should be another bindable for absolute / playfield-coordinate rotation

Reply:
There is, but I saw that only osu ruleset uses it, so it's inside the ruleset-specific rotate handler. If needed, I'll move it over to the generic rotate handler.

Reply:
the generic handler supports both modes so both bindables should be there.

Reply:
Fixed